kotlinpoet/src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -84,6 +88,10 @@ public class FunSpec private constructor(
       codeWriter.emitKdoc(kdoc.ensureEndsWithNewLine())
     }
     codeWriter.emitAnnotations(annotations, false)
+    if (contextReceiverTypes.isNotEmpty()) {
+      val receivers = contextReceiverTypes.joinToString(", ") { "%T" }
+      codeWriter.emitCode("context($receivers)\n", *contextReceiverTypes.toTypedArray())
```

Comment:
nit: alternative version
```suggestion
      val receivers = contextReceiverTypes
        .map { CodeBlock.of("%T", it) }
        .joinToCode(prefix = "context(", suffix = ")\n"))
      codeWriter.emitCode(receivers)
```